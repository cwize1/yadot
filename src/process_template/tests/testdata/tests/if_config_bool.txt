fruit:
- peach
- ${{ if .moreFruit }}:
    - banana
vegetable:
- carrot
- ${{ if .moreVegetables }}:
    - tomato
