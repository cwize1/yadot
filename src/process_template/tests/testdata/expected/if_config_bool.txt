ERROR: <None>
OUTPUT:
---
fruit:
  - peach
  - banana
vegetable:
  - carrot